templates/repo/diff/box.tmpl

```diff
@@ -1,5 +1,5 @@
 <div>
-	<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-ac gt-fw">
+	<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-fr gt-ac">
```

Comment:
I do not think it's right to have `flex-direction: column;` on `diff-detail-box` but now use another `!important` to override it.

These `gt-` helpers are only for some simple/inline/fine-tune purpose, they shouldn't be mixed with other explicit styles.

Reply:
Checking

Reply:
Fixed